src/library/browse/browsethread.h

```diff
@@ -49,7 +49,7 @@ class BrowseThread : public QThread {
 
     // You must hold m_path_mutex to touch m_path or m_model_observer
     QMutex m_path_mutex;
-    MDir m_path;
+    mixxx::FileAccess m_path;
```

Comment:
How about this:
```suggestion
    mixxx::FileAccess m_dir;
```

Reply:
I don't want to introduce many unrelated changes just by renaming things. And I especially try to avoid touching this browse stuff.